engine/src/main/java/com/datatorrent/stram/LocalAppHandle.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.datatorrent.stram;
+
+import com.datatorrent.api.Launcher;
+import com.datatorrent.api.LocalMode;
+
+/**
+ *
+ */
+public class LocalAppHandle implements Launcher.AppHandle
```

Comment:
Wanted to do that but the LocalModeImpl (EmbeddedAppLauncherImpl) is using the handle in the class declaration and the compiler does not like that to be a nested class